mangaki/mangaki/admin.py

```diff
@@ -304,6 +331,39 @@ class TaggedWorkAdmin(admin.ModelAdmin):
     search_fields = ('work', 'tag')
 
 
+@admin.register(WorkCluster)
+class WorkClusterAdmin(admin.ModelAdmin):
+    list_display = ('user', 'get_work_titles', 'resulting_work', 'reported_on', 'merged_on', 'checker', 'status')
+    raw_id_fields = ('user', 'works', 'checker', 'resulting_work')
+    actions = ('trigger_merge', 'reject')
+
+    def trigger_merge(self, request, queryset):
+        response = merge_works(request, queryset.first().works, queryset, from_cluster=True)
+        if response is None:
+            self.message_user(request, "La fusion %s a bien été effectuée." % queryset.first())
+        return response
+
+    trigger_merge.short_description = "Fusionner les œuvres de ce cluster"
+
+    def reject(self, request, queryset):
+        rows_updated = queryset.update(status='rejected')
+        if rows_updated == 1:
+            message_bit = "1 cluster"
+        else:
+            message_bit = "%s clusters" % rows_updated
+        self.message_user(request, "Le rejet de %s a été réalisé avec succès." % message_bit)
+
+    reject.short_description = "Rejeter les clusters sélectionnés"
+
+    def get_work_titles(self, obj):
+        if obj.works.exists():
+            return '<ul>' + ''.join('<li>%s (<a href="/admin/mangaki/work/%d/change/">%d</a>)</li>' % (work.title, work.id, work.id) for work in obj.works.all()) + '</ul>'
```

Comment:
Please use https://docs.djangoproject.com/en/dev/ref/utils/#django.utils.html.format_html_join and https://docs.djangoproject.com/en/dev/ref/contrib/admin/#reversing-admin-urls here.

```python
if obj.works.exists():
    def get_admin_url(work):
        return reverse('admin:mangaki_work_change', args=(work.id,))
    return (
        '<ul>' +
        format_html_join('', '<li>{} (<a href="{}">{}</a>)</li>',
            ((work.title, get_admin_url(work), work.id) for work in obj.works.all())) +
        '</ul>'
    )
else:
    return '(all deleted)'
```